crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -214,23 +211,23 @@ impl AcirContext {
         let result = if bit_size == 1 {
```

Comment:
nit (not blocking): In another PR we can remove these special case for booleans and just assume that the backend can handle these cases.

Reply:
This one was left in specifically because the backend isn't given enough context to identify the opportunity to optimise. (As of yet there is no OR blackbox.) Opened an issue in ACVM: https://github.com/noir-lang/acvm/issues/344